packages/react-magma-dom/src/components/Button/index.tsx

```diff
@@ -56,6 +56,11 @@ export interface ButtonStyles {
    * Defines the border radius
    * @default ButtonShape.fill
    */
+  /**
```

Comment:
Looks like you broke up the description of the `shape` prop with the prop itself.